_posts/2022-03-16-async-sink-base.md

```diff
@@ -0,0 +1,169 @@
+---
+layout: post
+title: "The Generic Asynchronous Base Sink"
```

Comment:
This blog does not cover/mention TableAPI. Is that intentional or a miss?

Reply:
Wanted to keep it succinct... I feel anyone well versed in building a connector, and has read through this intro, would be confident in creating the corresponding table api decorators.